### ISSUE TITLE
Markdown linting package and configuration file

### ISSUE THREAD

**Issue by jeff-matthews** [AdobeDocs/dev-site-documentation-template#80]
### Expected Behaviour

A markdown linting package, like [markdownlint](https://github.com/DavidAnson/markdownlint), that I can use to make sure my markdown files adhere to basic standards. Consistent markdown promotes usability for contributors.

I should be able to run linting locally as well as add it to an automated PR validation workflow (e.g., #79). 

The repo should include a default configuration file specifying a rule set at the root of the project. For example,

`.markdownlint.json`

```
{
  "default": false,
  "MD001": true,
  "MD003": { "style": "atx" },
  "MD004": true,
  "MD005": true,
  "MD006": true,
  "MD007": { "indent": 3 },
  "MD009": true,
  "MD010": true,
  "MD012": true,
  "MD018": true,
  "MD019": true,
  "MD023": true,
  "MD025": true,
  "MD029": true,
  "MD030": { "ol_single": 1, "ol_multi": 1, "ul_single": 2, "ul_multi": 2 },
  "MD032": true,
  "MD035": true,
  "MD036": true,
  "MD037": true,
  "MD038": true,
  "MD040": true,
  "MD046": { "style": "fenced"}
}
```

### Actual Behaviour

If I want to lint my markdown files, I have to add the package to my repo manually after it's been provisioned. It would be more efficient to add the linting package to the template repo so that it's available to all users who request a new repo.

### Reproduce Scenario (including but not limited to)

n/a

#### Steps to Reproduce

n/a

#### Platform and Version

n/a

#### Sample Code that illustrates the problem

n/a

#### Logs taken while reproducing problem

n/a

**Comment by jeff-matthews** [comment]
@timkim & @dmitrymatio,

If you think this would be useful for all template users, I'd be happy to create a PR. Just let me know.

**Comment by timkim** [comment]
Sounds good. The only way to lint right now is to run a build which isn't ideal. Send the PR and I'll close the issue once it's in.

**Comment by dshevtsov** [comment]
My suggestion also relates to https://github.com/AdobeDocs/dev-site-documentation-template/issues/79

**Comment by dshevtsov** [comment]
This issue can be closed by #81
